include/language-factory.php

```diff
@@ -98,6 +98,30 @@ public static function get_from_terms( array $terms ) {
 		$flag_props = self::get_flag( $data['flag_code'], $data['name'], $data['slug'], $data['locale'] );
 		$data       = array_merge( $data, $flag_props );
 
+		$default = array(
+			'home_url'   => '',
+			'search_url' => '',
+		);
```

Comment:
Do we really need this or can we just pass `array()` to the filter?

Reply:
This is used to restrict passed data with `array_intersect_key()` bellow.

Reply:
OK. Forget my last comment. I'd prefer to pass `array()` to the filter and use the array only to filter the output of `apply_filters()`.